### PR TITLE
[FIX] added missing elements and spike ins

### PR DESCRIPTION
**Pull request by timosachsenberg**
Added spike in compounds to database used in tutorial (only 7). Added some missing elements to database

### CODE REVIEW

File: share/OpenMS/CHEMISTRY/HMDBMappingFile.tsv [hunk 1]
```diff
@@ -1665,6 +1665,7 @@ database_version	3.5
 220.105921632	C8H16N2O5	HMDB:HMDB01104	HMDB:HMDB29071
 220.109944378	C13H16O3	HMDB:HMDB32589	HMDB:HMDB37492	HMDB:HMDB40424
 220.146329884	C14H20O2	HMDB:HMDB29675	HMDB:HMDB36389	HMDB:HMDB37606	HMDB:HMDB37714	HMDB:HMDB37718	HMDB:HMDB38996	HMDB:HMDB39539	HMDB:HMDB40217	HMDB:HMDB40226
+220.1502383412	C10(2)H3(1)H16NO4	EXTRA:EXTRA001
```
Comment (cbielow) [inline share/OpenMS/CHEMISTRY/HMDBMappingFile.tsv]: what are these?

**Comment by timosachsenberg** [comment]
pike in compounds to database used in tutorial.

**Comment by timosachsenberg** [comment]
EXTRA:EXTRA001  Propionyl-l-carnitine-d3        CCC(=O)OC(CC(=O)[O-])C[N+](C)(C)C([2H])([2H])[2H]       InChi=NA
EXTRA:EXTRA002  Nialamide       C1=CC=C(C=C1)CNC(=O)CCNNC(=O)C2=CC=NC=C2        InChi=NA
EXTRA:EXTRA003  Sulfadimethoxine-d6     C([2H])([2H])([2H])OC1=NC(=NC(=C1)NS(=O)(=O)C2=CC=C(C=C2)N)OC([2H])([2H])[2H]   InChi=NA
EXTRA:EXTRA004  Reserpine       CO[C@H]1[C@@H](C[C@@H]2CN3CCC4=C([C@H]3C[C@@H]2[C@@H]1C(=O)OC)NC5=C4C=CC(=C5)OC)OC(=O)C6=CC(=C(C(=C6)OC)OC)OC   InChi=NA
EXTRA:EXTRA005  Terfenadine     CC(C)(C)C1=CC=C(C=C1)C(CCCN2CCC(CC2)C(C3=CC=CC=C3)(C4=CC=CC=C4)O)O      InChi=NA
EXTRA:EXTRA006  Hexadecanoyl-l-carnitine-d3     CCCCCCCCCCCCCCCC(=O)O[C@H](CC(=O)[O-])C[N+](C)(C)C([2H])([2H])[2H]      InChi=NA
EXTRA:EXTRA007  Octadecanoyl-l-carnitine-d3     CCCCCCCCCCCCCCCCCC(=O)OC(CC(=O)[O-])C[N+](C)(C)C([2H])([2H])[2H]        InChi=NA

**Comment by timosachsenberg** [comment]
will add more elements to this pull request

**Comment by timosachsenberg** [comment]
by the way... thanks to @hannesveit and Marc for helping on the elements.xml

**Comment by timosachsenberg** [comment]
any objections?

**Pull request merged**